packages/svelte/svelte.mjs

```diff
@@ -0,0 +1,38 @@
+import { compile } from 'svelte/compiler';
+import { stripExtras } from '@nodejs-loaders/parse-filename';
+
+/**
+ * @type {import('node:module').ResolveHook}
+ */
+async function resolveSvelte(specifier, ctx, nextResolve) {
+	const nextResult = await nextResolve(specifier);
+
+	if (!stripExtras(specifier).endsWith('.svelte')) return nextResult;
```

Comment:
Clever :)

Reply:
I copy the logique that we used on css module